opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -66,25 +74,78 @@ def test_create(self):
         self.assertEqual(
             resource,
             resources._DEFAULT_RESOURCE.merge(
-                resources.Resource({resources.SERVICE_NAME: "unknown_service"})
+                resources.Resource(
+                    {resources.SERVICE_NAME: "unknown_service"}, ""
+                )
             ),
         )
+        self.assertEqual(resource.schema_url, "")
+
+        resource = resources.Resource.create(None, None)
```

Comment:
Is it necessary to have both the tests with
`resources.Resource.create(None, None)` and `resources.Resource.create(None)` in line 73?
Perhaps it could just be the first because of the tests after this one? I'm not too sure what's standard, so I could definitely be wrong!

Reply:
The tests have no knowledge about the default values if that's what you're concerned about. Aside from that, I'd rather be thorough and trim it away if the maintainers don't want clutter.